substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jdk/JDKLatest.java

```diff
@@ -28,9 +28,13 @@
 
 import jdk.graal.compiler.serviceprovider.JavaVersionUtil;
 
-public class JDK22OrEarlier implements BooleanSupplier {
+/**
+ * Denotes the latest supported JDK version. It corresponds to the highest key in the
+ * {@code JVMCI_MIN_VERSIONS} map in {@link jdk.graal.compiler.hotspot.JVMCIVersionCheck}.
+ */
+public class JDKLatest implements BooleanSupplier {
     @Override
     public boolean getAsBoolean() {
-        return JavaVersionUtil.JAVA_SPEC <= 22;
+        return JavaVersionUtil.JAVA_SPEC > 21;
```

Comment:
Nitpick: I understand that `21` is used here to avoid having to update the code with each STS JDK release, but this doesn't match the javadoc, nor represents the latest JDK.

Reply:
From my perspective, that is just an implementation detail. `21` is the most stable version we have and it only changes once every few years.